package.json

```diff
@@ -3,14 +3,15 @@
   "version": "2.0.0",
   "main": "./src/customfunctions.js",
   "scripts": {
-    "start": "start npm run dev-server && start npm run watch && npm run sideload",
-    "start-web": "start npm run dev-server && start npm run watch",
+    "start": "concurrently \"npm run dev-server\" && concurrently \"npm run watch\" && npm run sideload",
+    "start-web": "concurrently \"npm run dev-server\" && concurrently \"npm run watch\"",
```

Comment:
I tried the start-web command with this change on Windows and the dev-server kicks off but the watch command doesn't

Reply:
fixed with 607f0b5